### ISSUE TITLE
cbind's starting with a backslash (\) no longer work

### ISSUE THREAD

**Issue by GSI** [uzbl/uzbl#251]
Please paste the output of `uzbl-core --bug-info` here:

```
Commit: v0.9.0-17-g5af1c22
GTK compile: 3.18.9
GTK run: 3.18.9
WebKit compile: 2.4.10
WebKit run: 2.4.10
WebKit2: 0
libsoup compile: 2.52.2
libsoup run: 2.52.2
```

With the default config, nothing happens upon typing `\wiki`.

**Comment by mathstuf** [comment]
Saw this as well on one machine, but hadn't tracked it down. I do have an older binary/config/EM which is working, so theres one place to bisect.

**Comment by keis** [comment]
looks to be a case of escape inflation. a bind like `\\\\wiki` now works. So likely introduced by eccfa0730b4a7037cc3622b03b0ed547dc93f45b

**Comment by mathstuf** [comment]
I wonder if something like `@modmap "'" <Quote>` wouldn't work here as well.

**Comment by keis** [comment]
Yeah. I guess that could work. But I would rather keep moving towards consistent parsing of stuff in uzbl. Any command with split=FALSE and all the places in the eventmanager that calls out to `args.raw()` is making things much harder than they need to be.

Doing something like bcc0cd9a5107242439c156ac15ff70d1e066f807 should also fix this but has other issues. I dread to touch the parser of uzbl-core but if we could get rid of this bug that creates a lot of empty strings in the argument list I think it should just work (tm)

looks like this btw
```
ON_EVENT '' 'ESCAPE' '' '' '' 'event' 'KEYCMD_CLEAR'
```
